docs/src/content/hardhat-runner/docs/advanced/hardhat-and-foundry.md

```diff
@@ -0,0 +1,103 @@
+# Integrating with Foundry
+
+This guide explains how to use Hardhat and Foundry in the same project.
```

Comment:
This guide doesn't explain what the plugin does.

What's the expected end result here?

Reply:
I made some improvements in f0763bd8282ac998dbb8deb577cd87c275561b39